mlir/utils/performance/ck-benchmark-driver/CMakeLists.txt

```diff
@@ -1,17 +1,9 @@
-find_package(composable_kernel CONFIG 1.0.0 COMPONENTS device_operations)
-if(NOT composable_kernel_FOUND)
-  message(WARNING "Could not find composable-kernels, not building a CK benchmark driver")
-endif()
-
+find_package(composable_kernel 1.0.0 COMPONENTS device_operations CONFIG)
 if (composable_kernel_FOUND)
-  find_package(hip REQUIRED PATHS /opt/rocm)
-  set(LIBS composable_kernel::device_operations hip::device benchmark-driver-utils)
-  set(EXCLUDE_FROM_ALL TRUE)
```

Comment:
Please explain these changes 
Also, what's wrong with PATHS - if it's not in the path, it'll look somewhere else, no?